centos.org/pipelines/lib/foremanDuffyJob.groovy

```diff
@@ -30,6 +31,8 @@ pipeline {
         stage('Install Pipeline Requirements') {
             steps {
                 script {
+                    def duffy_session = readFile(file: 'jenkins-jobs/centos.org/ansible/duffy_session')
```

Comment:
What is this file?

Reply:
it gets written by provisionDuffy() to remember the session id, so that deprovisionDuffy() knows what box to drop